custom/icds_reports/models/aggregate.py

```diff
@@ -1590,6 +1592,33 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateAvailingServiceForms(models.Model, AggregateMixin):
+    """ Aggregated data for the availing services
+
+    A availing services exists for each state_id
+
+    A row exists for every person case that has had a record of registration
+    submitted against it this month
+    """
+    state_id = models.CharField(max_length=10)
```

Comment:
This broke the aggregation. State ids are UUIDs, well over 10 digits long. Why is this a max length char field? We generally prefer text fields for all text data (as you do for supervisor id and awc_id, but not person case) but a max_length 40 is also fine for these UUIDs. This table stores the same kind of data (a location id, or uuid) in 3 different column types, char fields of two different lengths and text fields. It is good to be consistent in storing the same type of data and in this case text fields are the safest.